app/src/main/java/org/p2p/wallet/bridge/claim/ui/ClaimPresenter.kt

```diff
@@ -154,7 +158,16 @@ class ClaimPresenter(
     }
 
     override fun onSendButtonClicked() {
-        // TODO implement claim logic
+        launch {
```

Comment:
if(latestBundle == null) return 

so no need to launch at all here
or make it with let
but do not launch and then check